meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -393,6 +396,9 @@ python mender_vars_handler() {
             json.dump(mender_vars, f, sort_keys=True, indent=4)
 }
 
+
+MENDER_DATA_PART_FSTAB_OPTS_append = "${MENDER_DATA_PART_FSTAB_OPTS_DEFAULT}${bb.utils.contains{'MENDER_FEATURES', 'mender-growfs-data' ',x-systemd.growfs', '', d)"
```

Comment:
Drop the `${MENDER_DATA_PART_FSTAB_OPTS_DEFAULT}` from above line,


though even when I did this locally I get a syntax error:

```
ERROR: /home/admin/hub/build/../sources/meta-raspberrypi/recipes-core/images/rpi-hwup-image.bb: Error during parse shell code, the last 5 lines are:
    fi

    mkdir -p /home/admin/hub/build/tmp/work/raspberrypi4-poky-linux-gnueabi/rpi-hwup-image/1.0-r0/rootfs/data
    printf "%-20s %-20s %-10s %-21s %-2s %s\n" ${tmpDataPart} /data auto defaults${bb.utils.contains{'MENDER_FEATURES', 'mender-growfs-data' ',x-systemd.growfs', '', d) 0 0 >> /home/admin/hub/build/tmp/work/raspberrypi4-poky-linux-gnueabi/rpi-hwup-image/1.0-r0/rootfs/etc/fstab
```

This probably means that we can not do this on `MENDER_DATA_PART_FSTAB_OPTS`

Reply:
haha, way ahead of me Mirza! Where should we do it then?

Reply:
It was your syntax, very subtle errors :),

This works as expected.

```
MENDER_DATA_PART_FSTAB_OPTS_append = "${@bb.utils.contains('MENDER_FEATURES', 'mender-growfs-data', 'x-systemd.growfs', '', d)}"
```

Reply:
haha, you're not supposed to do it all for me!